src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -91,79 +140,160 @@ const SidebarView = View.extend({
     info: '[data-info-region]',
     access: '[data-access-region]',
   },
+  initialize(options) {
+    this.mergeOptions(options, ['clinician', 'errors']);
+
+    this.listenTo(this.clinician, {
+      'change:_role': this.showInfo,
+      'change:_groups': this.showInfo,
+    });
+
+    this.state = new Backbone.Model();
```

Comment:
So there's nothing listening to this model, so I think perhaps this might be overcomplicating things.  if `showErrors(errors) {` just did `this.errors = errors` and inside `onSave` it can just be `this.errors = null` then there's no need for the `isEmpty` checks.  If we get an empty error we've got other problems.